modules/text2vec-aws/clients/aws.go

```diff
@@ -364,7 +364,8 @@ func (v *awsClient) getTargetVariant(config ent.VectorizationConfig) string {
 }
 
 type bedrockEmbeddingsRequest struct {
-	InputText string `json:"inputText,omitempty"`
+	InputText string    `json:"inputText,omitempty"`
+	Dimensions *int     `json:"dimensions,omitempty"`
```

Comment:
Let's change `Dimensions` type to `int64` to match the class's settings type (see my comment below)